### PR TITLE
Remove browser layer and registered javascripts / add extra javascript files from cycle2

### LINKED ISSUE #1: Center plugin is breaking fluid layouts
This plugin should be removed, and this kind of problem should be resolved into CSS or local JS.

This plugin added globally is breaking my fluid layout adding a margin-top into active slide.

screenshot from 2015-05-15 10 44 11

### PR DESCRIPTION
**Pull request by rodfersou**
closes #1

### CODE REVIEW

**Comment by hvelarde** [comment]
@rodfersou please add tests for the upgrade step

**Comment by rodfersou** [comment]
@hvelarde added upgrade step test

File: src/collective/js/cycle2/tests/test_upgrades.py [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+        title = u'Remove browser layer and jsregistry'
+        description = u'We don\'t need a profile for this package.'
+        step = self._get_upgrade_step(title)
+        self.assertIsNotNone(step)
+        self.assertEqual(step['description'], description)
+
+        self._do_upgrade_step(step)
+
+        js_tool = self.portal['portal_javascripts']
+        # simulate state on previous version
```
Comment (hvelarde) [inline src/collective/js/cycle2/tests/test_upgrades.py:67]: @rodfersou you are not simulating previous version anywhere here; you have to register the layer, the scripts and test they are indeed registered; then, after running the upgrade step, you will test they are no longer registered.

File: src/collective/js/cycle2/tests/test_upgrades.py [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+            '++resource++collective.js.cycle2/jquery.cycle2.center.min.js',
+            '++resource++collective.js.cycle2/jquery.cycle2.swipe.min.js'
+        ]
+
+        resource_ids = js_tool.getResourceIds()
+        for id in JS_IDS:
+            self.assertNotIn(id, resource_ids)
+
+        layers = [l.getName() for l in registered_layers()]
+        self.assertNotIn('collective.js.cycle2', layers)
```
Comment (hvelarde) [inline src/collective/js/cycle2/tests/test_upgrades.py]: @rodfersou this is no the right way to check this; see the test on the uninstall test case you just removed on test_setup.py

File: src/collective/js/cycle2/upgrades/profiles/default/browserlayer.xml [hunk 3]
```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0"?>
```
Comment (hvelarde) [inline src/collective/js/cycle2/upgrades/profiles/default/browserlayer.xml]: @rodfersou can we remove the layer programmatically? that way we have no need of registering a profile

Comment (rodfersou) [inline src/collective/js/cycle2/upgrades/profiles/default/browserlayer.xml]: done

File: src/collective/js/cycle2/tests/test_upgrades.py (hunk 1 above)
Comment (rodfersou) [inline src/collective/js/cycle2/tests/test_upgrades.py:67]: done

**Comment by rodfersou** [comment]
@hvelarde thank you for the code review

**Pull request merged**